spec/support/expect_offense.rb

```diff
@@ -1,168 +1,17 @@
-# frozen_string_literal: true
-
-# Mixin for `expect_offense` and `expect_no_offenses`
-#
-# This mixin makes it easier to specify strict offense expectations
-# and a declarative and visual fashion. Just type out the code that
-# should generate a offense, annotate code by writing '^'s
-# underneath each character that should be highlighted, and follow
-# the carets with a string (separated by a space) that is the
-# message of the offense. You can include multiple offenses in
-# one code snippet.
-#
-# @example Usage
-#
-#     expect_offense(<<-RUBY.strip_indent)
-#       a do
-#         b
-#       end.c
-#       ^^^^^ Avoid chaining a method call on a do...end block.
-#     RUBY
-#
-# @example Equivalent assertion without `expect_offense`
-#
-#     inspect_source(cop, <<-RUBY.strip_indent)
-#       a do
-#         b
-#       end.c
-#     RUBY
-#
-#     expect(cop.offenses.size).to be(1)
+# rubocop-rspec gem extension of RuboCop's ExpectOffense module.
 #
-#     offense = cop.offenses.first
-#     expect(offense.line).to be(3)
-#     expect(offense.column_range).to be(0...5)
-#     expect(offense.message).to eql(
-#       'Avoid chaining a method call on a do...end block.'
-#     )
-#
-# If you do not want to specify an offense then use the
-# companion method `expect_no_offenses`. This method is a much
-# simpler assertion since it just inspects the source and checks
-# that there were no offenses. The `expect_offenses` method has
-# to do more work by parsing out lines that contain carets.
+# This mixin is the same as rubocop's ExpectOffense except the default
+# filename ends with `_spec.rb`
 module ExpectOffense
+  include RuboCop::RSpec::ExpectOffense
```

Comment:
I'd protest if they do. We do have to explicitly `require 'rubocop/rspec/support'` to get it